# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse_capi/examples/dump.rs
# Commit: 00d82acfaa2f
# Full Hash: 00d82acfaa2f56095e5033a6905427d43b95eccb
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r 4ba88410cdf6 -r 00d82acfaa2f third_party/rust/mp4parse_capi/examples/dump.rs
--- a/third_party/rust/mp4parse_capi/examples/dump.rs	Fri Jan 27 18:35:14 2023 +0000
+++ b/third_party/rust/mp4parse_capi/examples/dump.rs	Fri Jan 27 18:35:14 2023 +0000
@@ -1,11 +1,4 @@
-extern crate mp4parse;
-extern crate mp4parse_capi;
-
-#[macro_use]
-extern crate log;
-
-extern crate env_logger;
-
+use log::info;
 use mp4parse::ParseStrictness;
 use mp4parse_capi::*;
 use std::env;
@@ -119,7 +112,9 @@
                         }
                     }
                 }
-                Mp4parseTrackType::Video => {
+                Mp4parseTrackType::Video
+                | Mp4parseTrackType::Picture
+                | Mp4parseTrackType::AuxiliaryVideo => {
                     let mut video_info = Mp4parseTrackVideoInfo::default();
                     match mp4parse_get_track_video_info(parser, i, &mut video_info) {
                         Mp4parseStatus::Ok => {